tests/test_warp.py

```diff
@@ -1293,7 +1321,39 @@ def test_reproject_resampling(path_rgb_byte_tif, method):
     assert np.count_nonzero(out) in expected[method]
 
 
-@pytest.mark.parametrize("test3d,count_nonzero", [(True, 1309625), (False, 437686)])
+@pytest.mark.parametrize(
+    "test3d,count_nonzero",
+    [
+        pytest.param(
+            True,
+            1309625,
+            marks=pytest.mark.skipif(
+                gdal_version_info >= (3, 10, 2), reason="See GDAL gh-11713"
```

Comment:
Here's the kernel of this PR. See https://github.com/OSGeo/gdal/issues/11711 and https://github.com/OSGeo/gdal/pull/11713.